docs/specs/oidc.md

```diff
@@ -300,24 +317,25 @@ To indicate the authenticator used in authentication, `amr` claim is used in OID
 - If password authenticator is used: `pwd` is included.
 - If any OTP (TOTP/OOB-OTP) is used: `otp` is included.
 - If WebAuthn is used: `hwk` is included.
+- If Biometric is used: `x_biometric` is included.
 
-If no authentication method is to be included in `amr` claim, `amr` claim would be omitted from the ID token.
+If no authentication method is included in `amr` claim, `amr` claim would be omitted from the ID token.
 
-### `acr`
+### `auth_time`
 
-If any secondary authenticator is performed, `acr` claim would be included in ID token with value `http://schemas.openid.net/pape/policies/2007/06/multi-factor`.
+The time when the authentication was performed.
```

Comment:
In what unit?

Reply:
[Unix epoch](https://openid.net/specs/openid-connect-core-1_0.html#IDToken). Let me be more specific here.